src/main/resources/application.yml

```diff
@@ -1,6 +1,6 @@
 logging:
   level.root: WARN
-  level.no.fdk: ${LOG_LEVEL:INFO}
+  level.no.fdk: ${LOG_LEVEL:DEBUG}
```

Comment:
```suggestion
  level.no.fdk: ${LOG_LEVEL:INFO}
```

Reply:
Kan det være lurt å ha på debug-logging når vi skal prodsette, for enklere og raskere finne ev. feil? Må selvfølgelig skru det av etterpå.